tests/opwen_email_server/integration/test_celery.py

```diff
@@ -10,6 +10,8 @@
 from kombu import Queue
 
 from opwen_email_server.config import QUEUE_BROKER
+from opwen_email_server.integration import celery as app  # ensure app tasks are loaded
+from celery import current_app  # cerely's __init__.py
```

Comment:
Is this import necessary? I assume we should be able to use `app.tasks` instead of `current_app.tasks` given that here `app` is already an instance of a Celery app.

Reply:
Correction: `from opwen_email_server.integration.celery import celery as app` would be an instance of the Celery app. What's currently being imported is a module (which renaming to `app` obscures somewhat).